src/main/java/lotto/LottoRankEnum.java

```diff
@@ -0,0 +1,20 @@
+package lotto;
+
+public enum LottoRankEnum {
+
+    FIRST(2000000000),
```

Comment:
```suggestion
    FIRST(2_000_000_000),
```
숫자의 경우 세자리 간격으로 `_`를 달아주시면 가독성에 도움이 됩니다 :)